cypress/integration/tests/controls/businessservices/crud.test.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress" />
+
+import { login } from "../../../../utils/utils";
+import { BusinessServices } from "../../../models/businessservices";
+import * as faker from "faker";
+
+describe("A single Business service", () => {
+    const businessService = new BusinessServices();
+
+    beforeEach("Login", function () {
+        // Perform login
+        login();
+    });
+
+    it("Business service crud operations", function () {
+        // Create new Business service
+        businessService.create();
+
+        businessService.exists();
+
+        // Edit Business service's name
+        businessService.edit({ name: faker.company.companyName() });
```

Comment:
FYI : You can call the model class method here, without affecting class var now and get rid of faker import.

`bussinessService.edit({ name: businessService.getBusinessServiceName() });`